apps/next/next.config.js

```diff
@@ -67,19 +67,22 @@ const withTM = require("next-transpile-modules")([
   "@showtime-xyz/universal.toast",
   "@showtime-xyz/universal.tooltip",
   "@showtime-xyz/universal.verification-badge",
-  "sentry-expo",
 ]);
 
 const isDev = process.env.NODE_ENV === "development";
 
 const nextConfig = {
+  swcMinify: true,
+  reactStrictMode: true,
   experimental: {
     optimizeCss: true,
-    esmExternals: "loose",
-    reactRoot: true,
+    browsersListForSwc: true,
     legacyBrowsers: false,
+    forceSwcTransforms: true,
+    esmExternals: true,
```

Comment:
I think this is the default value

Reply:
Ah yes haha we discussed this on Slack. Forgot to remove it (again)